huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go

```diff
@@ -71,7 +71,7 @@ func TestAccDmsRocketMQInstance_basic(t *testing.T) {
 				),
 			},
 			{
-				Config: testDmsRocketMQInstance_update(rName, updateName),
+				Config: testDmsRocketMQInstance_update(updateName),
 				Check: resource.ComposeTestCheckFunc(
 					rc.CheckResourceExists(),
 					resource.TestCheckResourceAttr(resourceName, "name", updateName),
```

Comment:
pls check the new attr: enterprise_project_id

Reply:
done

Reply:
done